tests/org.eclipse.n4js.n4ide.spec.tests/xpect-test/Ch03_02_02_08__QuickFix_Type_Issues/CONVERT_JAVA_TO_COLON_STYLE.n4js.xt

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2019 HAW Hamburg.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   HAW Hamburg - Initial API and implementation
```

Comment:
Please add your names here (you can also add your names to the other files below "Contributors" if you like.